### DISCUSSION TITLE
Having trouble uploading proof

### DISCUSSION THREAD

**Discussion by Sy0307** [community/community#130038] [GitHub Education]
### Hi everyone,

Hello Community!

I tried to re-verify my student status, I uploaded an 800KB image to verify my student status (the requirement on the webpage is for the image to be no larger than 1MB), but after I submitted it, it just refreshed the webpage and didn't report any errors or prompts. I found a "Payload too large" error in the console, so I resubmitted it with a compressed 400KB image and it worked.

I think this is a bug that you can try to reproduce. I think the "1MB" message in the page is incorrect.

**Comment by Valeriucoding** [comment]
I'm encountering the same issue, but my only option is to take a photo. Do you think there's a way to compress the actual photo that is taken?

**Reply by dmanolov** [reply]
I also was able to upload photo first time, my request was rejected because I forgot to fill my names, later when I fill them and tried again, it allows me only to take photo and not to upload one. I think this is a bug.

**Reply by Valeriucoding** [reply]
Currently it's working on the pc, but I don't have a good webcam. And when I try to do it on the phone it bugs out and reloads.

**Comment by taylortn** [comment]
Response #2
Response to #130038 

If you're having trouble uploading files to GitHub, whether it's for proof of work, documentation, or other purposes, there are several methods you can use. Here’s a step-by-step guide to help you upload your files to GitHub.

### Method 1: Using the GitHub Web Interface

1. **Navigate to Your Repository**:
   - Go to the repository where you want to upload your files.

2. **Upload Files**:
   - Click on the "Add file" button.
   - Select "Upload files".

3. **Drag and Drop or Choose Files**:
   - Drag and drop the files you want to upload, or click "choose your files" to browse and select the files from your computer.

4. **Commit Changes**:
   - Add a commit message describing your changes.
   - Click "Commit changes" to upload the files to your repository.

### Method 2: Using Git Command Line

1. **Clone the Repository**:
   - Open your terminal or Git Bash.
   - Clone your repository to your local machine:
     ```bash
     git clone https://github.com/username/repository.git
     ```
     Replace `username` and `repository` with your GitHub username and repository name.

2. **Navigate to the Repository Directory**:
   - Change to the directory of the cloned repository:
     ```bash
     cd repository
     ```

3. **Add Your Files**:
   - Copy or move the files you want to upload into the repository directory.
   - Add the files to the staging area:
     ```bash
     git add .
     ```

4. **Commit the Changes**:
   - Commit the files with a message:
     ```bash
     git commit -m "Add proof files"
     ```

5. **Push the Changes**:
   - Push the changes to your GitHub repository:
     ```bash
     git push origin main
     ```
     Replace `main` with the name of your branch if you are using a different branch.

### Method 3: Using GitHub Desktop

1. **Install GitHub Desktop**:
   - Download and install GitHub Desktop from [desktop.github.com](https://desktop.github.com/).

2. **Clone Your Repository**:
   - Open GitHub Desktop and sign in to your GitHub account.
   - Click on "Clone a repository from the Internet...".
   - Select your repository and clone it to your local machine.

3. **Add Your Files**:
   - Copy or move the files you want to upload into the local repository folder.

4. **Commit and Push**:
   - In GitHub Desktop, you will see your changes under the "Changes" tab.
   - Add a summary for your commit.
   - Click "Commit to main" (or your branch name).
   - Click "Push origin" to push the changes to GitHub.

### Troubleshooting Tips

- **File Size Limits**: GitHub has a file size limit of 100 MB for individual files. For larger files, consider using Git Large File Storage (LFS).
- **Permissions**: Ensure you have write permissions for the repository.
- **Branch**: Make sure you are pushing to the correct branch. By default, it's usually `main` or `master`.

Following these methods allows you to upload files to your GitHub repository successfully. If you encounter specific errors or issues, please provide more details so I can offer more targeted assistance.

**Reply by dmanolov** [reply]
Thank you for the answer! The problem is, that the web interface here: https://education.github.com/discount_requests/application does not allow any comment nor does have a single text box where I can put the URL of the document that I want to upload. The list of schools does not have my school, but I have no chance to report this problem too. :-)
